tests/src/integration/tests/test_ssl.cpp

```diff
@@ -170,6 +170,22 @@ CASSANDRA_INTEGRATION_TEST_F(SslTests, VerifyPeerIdentity) {
   write_and_read();
 }
 
+/**
+ * Ensures SSL connection verifying peer/server certificate while performing write and read
+ * operations.
+ */
+CASSANDRA_INTEGRATION_TEST_F(SslTests, VerifyPeerMultipleCerts) {
+  CHECK_FAILURE;
+
+  Ssl ssl;
+  ssl.with_verify_flags(CASS_SSL_VERIFY_PEER_CERT);
+  ssl.add_trusted_cert(SslCertificates::multi_cert_pem());
+
+  Cluster cluster = default_cluster().with_ssl(ssl);
+  connect(cluster);
+  write_and_read();
+}
```

Comment:
Simple integration test for the functionality added in [PR 522](https://github.com/datastax/cpp-driver/pull/522).